tests/test_execution.py

```diff
@@ -10,14 +10,11 @@
 
 
 @pytest.mark.slow
-def test_tuples_execution_on_same_node(global_execution_env):
+def test_tuples_execution_on_same_node(factory, client, data_envs_1):
```

Comment:
I think it would be good to follow a unique convention when defining fixtures? From "more common" fixture to "less common":
network, factory, client(s), data_env(s) ?

Reply:
That's what I tried to do. When you check the tests, you (almost) always need the `factory` fixture. The `network` is therefore used less than 10 times, because we directly import the profiles.